.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these owners will be
+# requested for review when someone opens a pull request.
+* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj @rapidray12 @schmelz1 @cdunn17 @aloftus23 @Matthew-Grayson @nickviola
+
+# These folks own any files in the .github directory at the root of
+# the repository and any of its subdirectories.
+/.github/ @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj
```

Comment:
```suggestion
/.github/ @rapidray12 @schmelz21 @cdunn17 @aloftus23 @Matthew-Grayson @nickviola
```

Reply:
@cduhn17 - please note the corrections. Thanks

Reply:
@cduhn17 noticed your handle was wrong too. Also we want to update the github managers too. Resubmitting some changes for you.